packages/cli/src/utils/misc.js

```diff
@@ -74,11 +74,8 @@ const runCommand = (command, args, options) => {
   });
 };
 
-const isValidNodeVersion = () => {
-  const v = semver(LAMBDA_VERSION);
-
-  return semver.satisfies(process.version, `>=${v.major}.${v.minor}`);
-};
+const isValidNodeVersion = (version = process.version) =>
+  semver.satisfies(version, `>=${semver.coerce(LAMBDA_VERSION)}`);
```

Comment:
`.coerce` plays nicely with `10.x` or whatever we need to put in as the lamba version. didn't end up needing it, but it's a good thing to use anyway.